frontend/src/app/modules/grids/widgets/time-entries-current-user/time-entries-current-user.component.ts

```diff
@@ -153,7 +153,12 @@ export class WidgetTimeEntriesCurrentUserComponent extends AbstractWidgetCompone
   }
 
   private formatNumber(value:number) {
-    return formatNumber(value, this.i18n.locale, '1.2-2');
+    try {
+      return formatNumber(value, this.i18n.locale, '1.2-2');
```

Comment:
/cc @ulferts  We cannot use angular formatting helpers without setting + loading the appropriate locale (which we could do with https://blog.angularindepth.com/dynamic-import-of-locales-in-angular-b994d3c07197

Reply:
Ah, thanks. I didn't know that.